scripts/install/lidarr.sh

```diff
@@ -37,40 +68,52 @@ cat > /home/${user}/.config/Lidarr/config.xml << LID
   <LaunchBrowser>False</LaunchBrowser>
 </Config>
 LID
-chown -R ${user}: /home/${user}/.config
-cat > /etc/systemd/system/lidarr.service << LID
+    mkdir -p /home/"${user}"/.tmp
+    chown -R "${user}": "/home/${user}/.tmp"
+    chown -R "${user}": "/home/${user}/.config"
+}
+systemd() {
+    cat > /etc/systemd/system/lidarr.service << LID
 [Unit]
-Description=lidarr for ${user}
+Description=Lidarr
 After=syslog.target network.target
 
 [Service]
 Type=simple
 User=${user}
 Group=${user}
 Environment="TMPDIR=/home/${user}/.tmp"
-ExecStart=/usr/bin/mono /home/${user}/Lidarr/Lidarr.exe -nobrowser
-ExecStop=-/bin/kill -HUP
+ExecStart=/opt/Lidarr/Lidarr -nobrowser -data=/home/${user}/.config/Lidarr/
```

Comment:
shouldn't this be LidarrOwner not User?

Reply:
nope, the user variable is populated at the top